lib/extensions/descendant_loader.rb

```diff
@@ -178,9 +178,15 @@ def cache
 
     def save_cache!
       return unless @cache_dirty
-      cache_path.parent.mkpath
-      cache_path.open('w') do |f|
-        YAML.dump(cache, f)
+
+      # Don't write sti_loader.yml in production, this shouldn't change from what is in the RPM
```

Comment:
If appliances don't already compile sti_loader as part of startup, maybe they should.  With that said, I don't know if this change breaks appliances.

Reply:
Nope, we moved to rpms a while ago... that shouldn't be an issue.